logical/framework/backend.go

```diff
@@ -612,6 +612,8 @@ func (t FieldType) Zero() interface{} {
 		return false
 	case TypeMap:
 		return map[string]interface{}{}
+	case TypeKeyPairs:
```

Comment:
Any thoughts on renaming this as `TypeKVPairs`?

Reply:
+1 from me on TypeKVPairs